tests/e2e/testPages/prebid-body.html

```diff
@@ -47,14 +47,14 @@
 					}
 				}
 			}
-			
+
```

Comment:
We should mute it by default

Reply:
OK - the mute setting is a changes to be made in BC Studio, not the code of this PR. So can we approve this and I'll make that change in studio today, @vpitts ?

Reply:
This change has been made in BC studio and is now live.